packages/apollo/src/commands/service/list.ts

```diff
@@ -101,37 +101,33 @@ export default class ServiceList extends ProjectCommand {
     const taskOutput: TasksOutput = {};
 
     let schema: GraphQLSchema | undefined;
+    let graphID: string | undefined;
+    let graphVariant: string | undefined;
     try {
       await this.runTasks<TasksOutput>(({ config, flags, project }) => {
-        if (!isServiceProject(project)) {
```

Comment:
This was for the sake of being able to run this command in a client project, right? I'm still not sure how I feel about allowing that from an ideological perspective, but technically, it's probably fine